packages/documentation/src/stories/foundation/accessibility/accessibility.docs.mdx

```diff
@@ -25,8 +27,94 @@ TODO: write accessibility guide about form labels (visible and invisible), aria-
+        </ul>
+        
+      </td>
+      <td>
+        <ul class="list-unstyled d-flex gap-mini">
+          <div>✔️</div><li>Let the user make mistakes and help them to correct them. Mark the required fields with error messages, scroll to the first invalid field and help the user fill the form correctly.</li>
+        </ul>
+       
+      </td>
+    </tr>
+    <tr>
+      <td colSpan="2" class="fw-bold">Disabled button on form submit</td>
+    </tr>
+    <tr>
+      <td>
+        <ul class="list-unstyled d-flex gap-mini">
+          <div>❌</div><li>The user submits a form. While the client waits for the server response and to prevent double submission, the submit button gets disabled. The reason why this button gets disabled might not be obvious.</li>
+        </ul>
+      </td>
+      <td>
+        <ul class="list-unstyled d-flex gap-mini">
+          <div>✔️</div><li>Provide a loading indicator or display a message that shows current progress when uploading large files. Disable duplicate form submissions internally. Inform users that the form was already submitted if they continue to click submit and offer a remedy if the request timed out or an error occurred on the server (reload the page, retry form submission, ...).</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td colSpan="2" class="fw-bold">Disabled select options</td>
+    </tr>
+    <tr>
+      <td>
+        <ul class="list-unstyled d-flex gap-mini">
+          <div>❌</div><li>Some options in a radio button group or select field are disabled because they are not available at this time or with the current configuration. This can be confusing because it's often unclear why these options are not available or how to enable them.</li>
+        </ul>
+      </td>
+      <td>
+        <ul class="list-unstyled d-flex gap-mini">
+          <div>✔️</div>
+          <li>
+          There are multiple possible solutions, depending on the situation:
+            <ul>
+              <li>Don't display options that cannot be selected and reduce clutter in your UI.</li>
+              <li>If you need to inform the user that there are potentially more possible options, show instructions on how to enable these options.</li>
+              <li>Display read-only data as text and not in disabled form fields.</li>
+            </ul>
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td colSpan="2" class="fw-bold">Form bad example</td>
+    </tr>
+    <tr>
+      <td><Canvas of={AccessibilityStories.FormBadExample}/></td>
+      <td><div class="mt-4"><Source code={formBadSample} language="ts" /></div></td>
+    </tr>
+     <tr>
+      <td colSpan="2" class="fw-bold">Form good example</td>
+    </tr>
+    <tr>
+      <td><Canvas of={AccessibilityStories.FormGoodExample}/></td>
+      <td><div class="mt-4"><Source code={formGoodSample} language="ts" /></div></td>
+    </tr>
```

Comment:
It's a bit sad how much space we loose, because of the fact, that the actual form and the validator code example are shown in cells beside each other:
![image](https://github.com/swisspost/design-system/assets/9716662/b88a1c88-9d3c-49b1-a3ab-d2486bb91408)

1. Remove the `<div class="container">...</div>` element which wraps the form.
2. Try to place the good and bad examples beside each other.
3. Try to add the validator code example into a script tag, within the HTML example, so it's only visible if the user expandes the code from story-panel.
I'm not even sure, if the visualisation of this code adds any value to the example. Consider removing them completely, as DEVs should understand the point, by playing around with the story examples itself.

```html
<tr>
  <td class="fw-bold">Form bad example</td>
  <td class="fw-bold">Form good example</td>
</tr>
<tr>
  <td><Canvas of={AccessibilityStories.FormBadExample}/></td>
  <td><Canvas of={AccessibilityStories.FormGoodExample}/></td>
</tr>
```

If the above changes do not lead to a more proper visualisation, try to move the examples out of the table and add them to the HTML directly after the table.

Reply:
I think we are still not there.
Can you please move the "Form bad" and "Form good" examples out of the table and implement one after another?

And by the way, please add the `novalidate` attribute to the form tags, to prevent native error messages show up.